djimporter/importers.py

```diff
@@ -130,10 +130,18 @@ def is_valid(self):
         if self.errors:
             return False
 
+        num_lines = len(self.csv_file) - 1
+        # Status progress will be saved 10 times
+        block_lines = int(num_lines / 10)
```

Comment:
If num_lines < 10 , block_lines will be 0; so line 140 will raise ZeroDivision error.

Reply:
Fixed setting block_lines = 1 when num_lines < 10. In this case there is only 1 block of lines.